src/alda/AldaClient.java

```diff
@@ -68,8 +73,9 @@ public static void updateAlda() throws URISyntaxException {
     }
 
     if (downloadURL == null) {
-      System.err.println("Alda download link not found for your platform.");
-      return;
+      throw new UnsuccessfulException(
+        "Alda download link not found for your platform."
```

Comment:
That's a good point. I can add something like "Please file an issue at $github_link_goes_here."